### ISSUE TITLE
Feature request

### ISSUE THREAD

**Issue by Jamesgresley** [wseemann/RoMote#23]
I would like to be able to set my preferred starting page/tab instead of always starting with devices. 

Also  I would like to be able to change a setting to stop seeing the power off confirmation.

Thanks much for a great app.

**Comment by e-t-l** [comment]
Yes, especially when there is only one device connected, opening the app should go immediately to the Remote tab (if not just letting users configure the "default" tab to begin with)

**Comment by pupitetris** [comment]
> Yes, especially when there is only one device connected, opening the app should go immediately to the Remote tab (if not just letting users configure the "default" tab to begin with)

I think this has already been added in the latest version.

**Comment by wseemann** [comment]
This is fixed in the latest [release](https://github.com/wseemann/RoMote/releases/tag/v2.1.0).
